solace-spring-boot-autoconfigure/solace-java-spring-boot-autoconfigure/src/main/java/com/health/indicators/SolaceFlowHealthIndicator.java

```diff
@@ -0,0 +1,25 @@
+package com.health.indicators;
+
+import com.health.base.SolaceHealthIndicator;
+import com.solacesystems.jcsmp.FlowEventArgs;
+import org.springframework.lang.Nullable;
+
+public class SolaceFlowHealthIndicator extends SolaceHealthIndicator {
+	public void up() {
+		super.healthUp();
+	}
+
+	public void reconnecting(@Nullable FlowEventArgs eventArgs) {
+		super.healthReconnecting(eventArgs);
+	}
+
+	public void down(@Nullable FlowEventArgs eventArgs) {
+		super.healthDown(eventArgs);
+	}
+
+	@Deprecated
+	public void down(@Nullable FlowEventArgs eventArgs, boolean resetReconnectCount) {
+		super.healthDown(eventArgs);
+	}
```

Comment:
I suggest creating a sub-class of `SolaceFlowHealthIndicator` in the Solace SCSt binder for this method. `resetReconnectCount` is a concept we defined in the Solace SCSt binder. It isn't a concept we have in Solace Spring Boot.